py-polars/tests/unit/test_lazy.py

```diff
@@ -1283,6 +1283,14 @@ def test_max_min_multiple_columns(fruits_cars: pl.DataFrame) -> None:
     assert res.to_series(0).series_equal(pl.Series("min", [1, 2, 3, 2, 1]))
 
 
+def test_max_min_wildcard_columns(fruits_cars: pl.DataFrame) -> None:
+    res = fruits_cars.select([pl.col(pl.datatypes.Int64)]).select(pl.min(["*"]))
+    assert res.to_series(0).series_equal(pl.Series("min", [1, 2, 3, 2, 1]))
+
+    res = fruits_cars.select([pl.col(pl.datatypes.Int64)]).select(pl.max(["*"]))
```

Comment:
Could we test this with `pl.all()` as well? That is the ideomatic way to express that you want all columns.